### DISCUSSION TITLE
Armv7 support

### DISCUSSION THREAD

**Discussion by RubenTeixeira** [jeffvli/feishin#339] [Q&A] [answered]
I'd like to try this on my Rpi4. Is there any chance this will build on armv7?

Thanks!

**Comment by jeffvli** [comment, marked as answer]
Yes, #336 was just merged recently which will include armv7 builds for future releases.

**Reply by RubenTeixeira** [reply]
Awesome! Can't wait to try this out, especially when new OpenSubsonic features become available.
Thanks for your work.
